pkg/cmd/addon/disable/cmd.go

```diff
@@ -47,6 +47,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 
 	cmd.Flags().StringSliceVar(&o.names, "name", []string{}, "Names of the add-on to deploy (comma separated)")
 	cmd.Flags().StringSliceVar(&o.clusters, "cluster", []string{}, "Names of the managed cluster to deploy the add-on to (comma separated)")
+	cmd.Flags().BoolVar(&o.allclusters, "all-clusters", false, "Make all maanged clusters to disable the add-on")
```

Comment:
type maanged

otherwise looks good

/approve

Reply:
so sorry for my caleless...edited it and the examples https://github.com/open-cluster-management-io/clusteradm/pull/115/files#diff-ce8c5a42db5d6eaba36ade94a6c4420c6c152bd06a92e3489f306e3e0db56624R15-R18